pilot/pkg/model/validation_test.go

```diff
@@ -550,10 +574,14 @@ func TestValidateMeshConfig(t *testing.T) {
 	} else {
 		switch err.(type) {
 		case *multierror.Error:
-			// each field must cause an error in the field
-			if len(err.(*multierror.Error).Errors) < 6 {
```

Comment:
Before my changes there were actually 12 errors, so adding 3 more didn't make this fail. Instead of checking the count I refactored this test to actually compare the error strings.